framework/Models/Code.py

```diff
@@ -338,7 +338,7 @@ def createNewInput(self,currentInput,samplerType,**kwargs):
 
     return (newInput,kwargs)
 
-  def __expandForWindows(self, origCommand):
+  def _expandForWindows(self, origCommand):
```

Comment:
This doesn't seem to be used outside the class, so is there a reason it was switched from private to protected?

Reply:
As a private member it wasn't recognizing it when called later on.  I don't know why that is, but this change fixed it.

Reply:
Okay.